ci/licenses_golden/excluded_files

```diff
@@ -166,6 +166,8 @@
 ../../../flutter/impeller/geometry/size_unittests.cc
 ../../../flutter/impeller/geometry/trig_unittests.cc
 ../../../flutter/impeller/golden_tests/README.md
+../../../flutter/impeller/golden_tests_harvester/.dart_tool
```

Comment:
These shouldn't have been added back. There might be some stale files in the bot's tree, maybe?